README.md

```diff
@@ -1 +1,13 @@
-# hw01
\ No newline at end of file
+
+
+### My biography
+
+My name is **Guillermo Guevara** and I am second year MPP at Harris. Before coming to Harris, I held several positions at the Mexican Federal Government.I am interested in:
+
+- Policing
+- Crime prevention
+- Political Accountability
+
+[Here you can check a photo of my idol Andy Murray](https://en.wikipedia.org/wiki/Andy_Murray#/media/File:Andy_Murray_(44087043305).jpg)
+
```

Comment:
I didn't think of embedding an image link in the homework, so this was an interesting choice. On an aesthetic level, I think it fits the concise and clear style of your biography. On the other hand, I do think that embedding an image itself would have been better for the assignment.